upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -921,25 +899,6 @@ There you can find links to upgrade notes for other versions too.
     +       tags:
     +           - { name: shopsys.cron, hours: '*', minutes: '*' }
     ```
-
-- update FpJsFormValidator bundle ([#1664](https://github.com/shopsys/shopsys/pull/1664))
-    - update your `composer.json`
-      ```diff
-            "require": {
-      -         "fp/jsformvalidator-bundle": "^1.5.1",
-      +         "fp/jsformvalidator-bundle": "^1.6.1",
-            }
-      ```
-    - update your `.eslintignore`
-      ```diff
-        /assets/js/commands/translations/mocks
-      + /assets/js/bundles
-      ```
-    - update your `.gitignore`
-      ```diff
-        /assets/js/translations.json
-      + /assets/js/bundles
-      ```
 - update your application to support multiple delivery addresses ([#1635](https://github.com/shopsys/shopsys/pull/1635))
```

Comment:
Shouldn't be these instructions about multiple delivery addresses together with instructions about changing user structure ?

Reply:
No, these are two different changes